.gitignore

```diff
@@ -58,6 +58,5 @@ test/functional/*/*/test.log
 test/functional/*/*/debug.log
 test/functional/*/*/state
 test/functional/*/*/lines-output
-test/functional/*/*/target-dir/usr/share/clear/bundles/.MoM
```

Comment:
You also removed this line in #716... just a warning that either this PR or that one will need a rebase :)